libraries/WiFi/src/WiFiGeneric.h

```diff
@@ -103,6 +103,16 @@ class WiFiGenericClass {
   bool setSleep(wifi_ps_type_t sleepType);
   wifi_ps_type_t getSleep();
 
+  /**
+  * @brief Set modem sleep state
+  * @param sleepType Modem sleep type, one of WIFI_PS_NONE, WIFI_PS_MIN_MODEM, WIFI_PS_MAX_MODEM
+  * @param listenInterval Listen interval for ESP32 station to receive beacon when WIFI_PS_MAX_MODEM is set. Units: AP beacon intervals. Defaults to 3 if set to 0.
```

Comment:
it would be good to indicate the units of this parameter. As written it could be interpreted as milliseconds, seconds, minutes, etc.

Reply:
Unit is specified here (AP beacon intervals). It is determined by the access point parameters, I've often seen a 102400us value during testing with different routers, however it can differ.